tests/cpptests/test_cpp_value.cpp

```diff
@@ -15,56 +15,58 @@ class ValueTest : public ::testing::Test {};
 
 TEST_F(ValueTest, testBasic) {
```

Comment:
If I get picky we could add more unittests for the new APIs, but since it is a temp step towards Rust code, I guess we can aim to have all that there.

Reply:
I would indeed suggest adding those unit tests to the Rust code instead